include/tensor.tcc

```diff
@@ -113,140 +136,154 @@ void* Tensor<T>::data_view(const MemoryLayout mem_layout)
        of the row major layout.
     */
 
-    void* ptr = 0;
+    void* ptr = NULL;
 
-    switch(mem_layout) {
-        case(MemoryLayout::contiguous) :
-            ptr = this->_data;
+    switch (mem_layout) {
+        case MemoryLayout::contiguous:
+            ptr = _data;
             break;
-        case(MemoryLayout::fortran_contiguous) :
-            ptr = this->_f_mem_views.allocate_bytes(
-                                     this->_n_data_bytes());
-            this->_c_to_f_memcpy((T*)ptr, (T*)this->_data, this->_dims);
+        case MemoryLayout::fortran_contiguous:
+            ptr = _f_mem_views.allocate_bytes(
```

Comment:
Probably fits on a single line of text without ``this``.

Reply:
ok